specs/validator/tableland-openapi-spec.yaml

```diff
@@ -25,6 +25,7 @@ externalDocs:
   url: http://docs.tableland.xyz
 servers:
   - url: https://testnets.tableland.network/api/v1
+  - url: http://localhost:8080/api/v1
```

Comment:
Really handy for local testing. Can use a VSCode plugin to test the APIs right from inside VSCode!